xrpl/asyncio/transaction/main.py

```diff
@@ -142,6 +143,30 @@ async def submit_transaction(
     raise XRPLRequestFailureException(response.result)
 
 
+@deprecated(
```

Comment:
This doesn't need to be deprecated until v2.0 (if then). It can stay as a supported alias.